tox.ini

```diff
@@ -150,6 +150,22 @@ commands=
 
   testinfra -n 4 --sudo -v --connection=ansible --ansible-inventory={changedir}/hosts-switch-to-containers {toxinidir}/tests/functional/tests
 
+[add-mons]
+commands=
+  ansible-playbook -vv -i {changedir}/hosts-2 --limit mons {toxinidir}/tests/functional/setup.yml
+  cp {toxinidir}/infrastructure-playbooks/add-mon.yml {toxinidir}/add-mon.yml
+  ansible-playbook -vv -i {changedir}/hosts-2 --limit mons {toxinidir}/add-mon.yml --extra-vars "\
```

Comment:
the --limit here is not necessary since you already target a group of node in add-mon.yml